doc/variableexplorer.rst

```diff
@@ -2,95 +2,235 @@
 Variable Explorer
 #################
 
-The **Variable Explorer** shows the namespace contents (all global object references, such as variables, functions, modules, etc.) of the currently selected :doc:`ipythonconsole` session, and allows you to interact with them through a variety of GUI-based editors.
+The **Variable Explorer** allows you to interactively browse and manage the variables and objects generated running your code.
 
-.. image:: images/variable_explorer/variable_explorer_standard.png
+.. image:: images/variable_explorer/variable-explorer-execution.gif
+   :width: 500
    :align: center
-   :alt: Spyder Variable Explorer, with a list of variables and their contents
+   :alt: Spyder Variable Explorer execution with a variable of type list
 
 |
 
+It shows the namespace contents (including all global objects, variables, class instances and more) of the currently selected :doc:`ipythonconsole` session, and allows you to add, remove, and edit their values through a variety of GUI-based editors.
+
+.. image:: images/variable_explorer/variable-explorer-standard.png
+   :align: center
+   :alt: Spyder Variable Explorer, with a list of variables and their contents
 
-====================
-Features and editors
-====================
+|
 
-Spyder's :guilabel:`Variable Explorer` offers built in support for editing lists, strings, dictionaries, NumPy arrays, Pandas DataFrames, and more, and can also histogram, plot, or even display some of them as an RGB image.
-Several examples of this functionality follow:
+The Variable Explorer gives you information on the name, size, type and value of each object. To modify a scalar variable, like an number, string or boolean, simply double click it in the pane and type its new value.
+
+.. image:: images/variable_explorer/variable-explorer-modifying.gif
```

Comment:
This one shows a visual bug in Spyder: the spinner never stops. I think the problem is that the signal to tell it stop is never emitted when editing inline objects. I'll fix it on the Spyder side.

Reply:
Done

Reply:
LGTM

Reply:
Thanks @juanis2112 for fixing the bug!